README.md

```diff
@@ -76,6 +76,17 @@ MyType.prototype[fl.map] = MyType.prototype.map
 var foo = bar[fl.map](x => x + 1)
 ```
 
+## Type representatives
+
+Certain behaviours are defined from the perspective of a member of a type.
+Other behaviours do not require a member. Thus certain algebras require a
+type to provide a value-level representative (with certain properties). The
+Identity type, for example, could provide `Id` as its type representative:
+`Id :: TypeRep Identity`.
```

Comment:
In Haskell types are "real" in the sense that one actually references types in one's programs:

```haskell
inc :: Int -> Int
inc x = x + 1
```

In JavaScript types are not referenced (except in comments):

```javascript
//    inc :: Number -> Number
const inc = x => x + 1;
```

In JavaScript the *idea* of types exists—as evidenced by the use of type signatures in comments—but there's no "type level"; only a "value level". Since we can't reference types at the type level, we must reference them at the value level. So, for each type we'd like a value which represents it. We call such values type representatives.

Let's consider some examples:

  - `Number` is the representative of the Number type. `Number` is the sole inhabitant of the `TypeRep Number` type.

  - `Id` is the representative of the Identity type. `Id` is the sole representative of the `TypeRep Identity` type.

The formatting above is very much intentional. The Number type exists only as an idea, so should not be formatted as code. `Number`, on the other hand, is a JavaScript value which represents the Number type (in addition to its role as a function from `Any` to `Number`).

It took me a while to come to terms with the idea of type representatives. This sort of thinking is not natural in a dynamically typed language. I suggest reading sanctuary-js/sanctuary#64 if you have not already done so.